lib/AcqUnits/hooks/useAcqRestrictions/useAcqRestrictions.js

```diff
@@ -0,0 +1,80 @@
+import { useQuery } from 'react-query';
+import { useStripes, useOkapiKy } from '@folio/stripes/core';
+
+const useAcqMemberships = (entityId, acqUnits = []) => {
+  const ky = useOkapiKy();
+  const stripes = useStripes();
+  const userId = stripes.user?.user?.id;
+
+  const searchParams = {
+    limit: 1000,
+    query: `userId=${userId} and (${acqUnits.map(id => `acquisitionsUnitId=${id}`).join(' or ')})`,
```

Comment:
isn't that a problem - length of acqUnits ? if too many of them

Reply:
we do the same for field and view, seems no